### ISSUE TITLE
Block access to mentoring if you have no slots remaining

### ISSUE THREAD

**Issue by Eddie-Hartman** [exercism/docs#250]
After the before the upgrade to v3, I had 4 exercises waiting for mentor feedback. After the upgrade, I planned on putting them back into the mentor queue, but was hit with
>You have no more mentoring slots available.
You can get more mentoring slots by earning reputation.

but didn't see how much reputation would net me more slots. So I think this prompts the addition of a section for showing what other potential rewards (other than badges) may be out there for those who gather a lot of reputation. I think it's a good motivator for gathering reputation.

At minimum, there should probably be a link explaining how much reputation is needed for more mentoring slots.

**Comment by JaPrad** [comment]
Hello @Eddie-Hartman!
The no. of reputations you need to unlock next slot must be visible just above your mentoring slot

Consider this from my javascript track:
![image](https://user-images.githubusercontent.com/73636668/132717436-8ec8677c-cdae-4339-a425-178984012fc3.png)
You must be seeing `Next slot unlocks at 100 rep` in the above screenshot

Does your interface display something else?
Could you share a screenshot, and specify which track you are in, if so? That would help other contributors.

**Comment by Eddie-Hartman** [comment]
Just to be clear, I'm not asking about me being a mentor, but for getting mentored. I don't know where you went to see the above, but the point kind of being that I was presented with this screen, but there is no info on how much reputation I need to unlock another slot or a link to see. Just a link to the wiki on what reputation is. I see this when I try to request mentoring on another exercise:
![image](https://user-images.githubusercontent.com/14089997/132729381-8103aa10-6d68-4485-bfcf-303478394963.png)


This is what I see on my track:
![image](https://user-images.githubusercontent.com/14089997/132729326-bccecf66-5d3d-4f80-9ee3-895e4620379e.png)

**Comment by iHiD** [comment]
@Eddie-Hartman Scroll down the track page a little and you'll see what @JaPatGitHub is talking about.

**Comment by Eddie-Hartman** [comment]
@JaPatGitHub @iHiD Perfect! Thank you.

For closing this, I think it would simply be beneficial to see that section on the page where you've ran out of slots so that you can see the currently mentored exercises and how much reputation to unlock the next slot.

Sorry for the runaround. Still getting used to the new site.

**Comment by iHiD** [comment]
Yep - that makes sense. The actual page itself is temporary - all buttons about requesting mentoring should be disabled with a tooltip that explains this. So I'll remember to add something there. I'll leave this open and rename it to be about this :)

**Comment by ErikSchierboom** [comment]
Is this a docs issue or more of a website issue?

**Comment by Eddie-Hartman** [comment]
@ErikSchierboom 
What I was calling out definitely called attention to both. From the website side, it taking me to a "Whoops" page was unexpected because the link it provided just took you to a generic reputation page that didn't actually explain how much reputation I would need to unlock another slot or how to find that information.

For the docs side, seeing: "You can get more mentoring slots by earning reputation.", but then not explaining at all how that system worked, how much reputation you needed to get more slots, etc. left me hanging a bit.

I had to go to the track page then scroll down to see the info I was looking for, but it wasn't in the actual mentoring and reputation sections or in the docs where I would have expected since I went looking for it. Even a mention in the docs to look at the track page to see how much reputation you need to unlock more slots would have sufficed, but I didn't know where to go at the time.
